webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Button } from 'patternfly-react';
+import { syncInsights } from './InsightsCloudSyncActions';
```

Comment:
you will receive the actions through component props, after the component will be connected the redux.
can you connect it in the `index.js` file?
as a reference you can copy the [AccountList/index.js](https://github.com/theforeman/foreman_rh_cloud/blob/master/webpack/ForemanInventoryUpload/Components/AccountList/index.js)